SWLOR.Game.Server/Feature/PerkDefinition/ForcePerkDefinition.cs

```diff
@@ -344,23 +344,21 @@ private void ForceBurst()
         }
         private void ForceMind()
         {
-            _builder.Create(PerkCategoryType.ForceLight, PerkType.ForceMind)
+            _builder.Create(PerkCategoryType.ForceUniversal, PerkType.ForceMind)
                 .Name("Force Mind")
 
                 .AddPerkLevel()
-                .Description("Converts 25% of the user's FP into HP.")
+                .Description("Grants Stamina Point regeneration based on your Willpower Score, at the cost of your Willpower.")
```

Comment:
I like the addition of the scaling info but we should probably be consistent with naming. "STM" is used elsewhere so I would replace "Stamina Point" with "STM" or just "Stamina" since we don't ever make a reference to points to my knowledge.